src/core_tests.rs

```diff
@@ -29,7 +30,7 @@ use node::Node;
 use peer_manager::{MIN_GROUP_SIZE, QUORUM_SIZE};
 use rand::{self, Rng, SeedableRng, XorShiftRng};
 use rand::distributions::{IndependentSample, Range};
-use routing_table::{RoutingTable, Xorable};
+use routing_table::{RoutingTable, Xorable, verify_network_invariant};
```

Comment:
Just another nitpick - we generally don't fully import functions via `use`, only structs, enums, and constants I think.  This should probably have been `self` so we can then use `routing_table::verify_network_invariant()`.